sycl/include/sycl/info/ext_codeplay_device_traits.def

```diff
@@ -2,6 +2,7 @@
 #define __SYCL_PARAM_TRAITS_TEMPLATE_SPEC_NEEDS_UNDEF
 #define __SYCL_PARAM_TRAITS_TEMPLATE_SPEC __SYCL_PARAM_TRAITS_SPEC
 #endif
+// TODO(#15184): Remove the fusion aspect in the next ABI-breaking window.
```

Comment:
#15185

Reply:
No, I've created an issue to track this in #15184.